Source/Core/Core/Boot/ElfReader.h

```diff
@@ -39,7 +39,7 @@ class ElfReader
   ElfMachine GetMachine() const { return (ElfMachine)(header->e_machine); }
   u32 GetEntryPoint() const { return entryPoint; }
   u32 GetFlags() const { return (u32)(header->e_flags); }
-  bool LoadIntoMemory();
+  bool LoadIntoMemory(bool only_in_mem1 = true);
```

Comment:
Whoops, this should default to false… Good catch, thanks! 0x1800000 is the mem1 size; I'll make it a proper constant.